lua/lspconfig/perlls.lua

```diff
@@ -0,0 +1,37 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.perlls = {
+  default_config = {
+    cmd = {"perl",
+    "-MPerl::LanguageServer",
+    "-e", "Perl::LanguageServer::run","--",
+    "--port 13603", "--nostdio 0", "--version 2.1.0"};
+    settings = {
+      perl = {
+        perlCmd = 'perl';
+        perlInc = ' ';
+        fileFilter = {".pm",".pl"};
+        ignoreDirs =  '.git';
+      };
+    };
+    filetypes = {"perl"};
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname) or vim.loop.os_homedir()
```

Comment:
Do you mean to set the root_dir to home here as opposed to the directory containing the file?

Reply:
aiya, you're right, changed to vim.fin.getcwd()